README.md

```diff
@@ -1,3 +1,4 @@
+#mahdi malvandi
```

Comment:
You should not add my name in the read me. 
Instead try adding `Changiz discord bot` to increase the readability